packages/mobile/src/firebase/remoteConfigValuesDefaults.e2e.ts

```diff
@@ -49,4 +49,5 @@ export const REMOTE_CONFIG_VALUES_DEFAULTS: Omit<
   dappListApiUrl:
     'https://raw.githubusercontent.com/valora-inc/dapp-list/main/translations/valora-dapp-list-base.json',
   superchargeButtonType: SuperchargeButtonType.PillRewards,
+  recentDappsEnabled: false,
```

Comment:
i enabled it for the e2e tests now since there's no harm, but initially i left it out because this feature sounded very experimental and likely to move somewhere else in the app from my discussion with Nitya :)